server/src/main/java/au/org/aodn/ogcapi/server/core/service/ElasticSearch.java

```diff
@@ -69,15 +71,28 @@ protected BoolQuery createBoolQueryForProperties(List<Query> must, List<Query> s
         return builder.build();
     }
 
-    protected List<StacCollectionModel> searchCollectionBy(List<Query> queries, List<Query> should, List<Query> filters, Integer from, Integer size) throws IOException {
+    protected List<StacCollectionModel> searchCollectionBy(List<Query> queries,
+                                                           List<Query> should,
+                                                           List<Query> filters,
+                                                           List<String> property,
```

Comment:
will `properties` makes it clearer that the param means more than 1 thing to be returned?